docs/StardustDocs/topics/operations.md

```diff
@@ -3,8 +3,8 @@
 <!---IMPORT org.jetbrains.kotlinx.dataframe.samples.api.Modify-->
 
 Data transformation pipeline usually consists of several modification operations, such as filtering, sorting, grouping, pivoting, adding/removing columns etc. 
-DataFrame API is designed in functional style so that the whole processing pipeline can be represented as a single statement with a sequential chain of operations.
-`DataFrame` object is immutable and all operations return a new [`DataFrame`](DataFrame.md) instance reusing underlying data structures as much as possible.
+Data frame API is designed in functional style so that the whole processing pipeline can be represented as a single statement with a sequential chain of operations.
```

Comment:
I think this first is supposed to mean the library, so "The Kotlin DataFrame API", or "The Kotlin DataFrame library API"